### PR TITLE
Component to Show or Hide the content 

### PR DESCRIPTION
**Pull request by sudhacheran**
- New component added to show to hide its content based in the 'visible' option provided.
- Installation workflow name changed as per the request.

### CODE REVIEW

**Review by charliepnnl — CHANGES_REQUESTED** [review]
Just a few changes needed.

File: src/components/show_or_hide.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    children: ReactNode;
+    label: string;
+    visible: boolean;
+}
+
+/**
+ * A component that conditionally shows or hides its children based on the 'visible' prop.
+ * If 'visible' is true, it renders the 'children'; otherwise, it renders nothing.
+ */
+const ShowOrHide: FC<ShowOrHideProps> = ({ children, label, visible }) => {
```
[the remark this answers is not in the document]
Comment (sudhacheran) [reply src/components/show_or_hide.tsx]: Removed the 'label' prop

File: src/templates/doe_workflow_central_ducted_split_heat_pump.mdx [hunk 2]
```diff
@@ -314,23 +335,47 @@
...
+                <p><strong>Conditioned Floor Area (ft<sup>2</sup>) = </strong>{props.data.cfm25_calculator?.conditioned_floor_area ? props.data.cfm25_calculator.conditioned_floor_area : null} </p>
+                <p><strong>Duct CFM25 per 100 per ft<sup>2</sup> = </strong>{ props.data.cfm25_calculator?.cfm25 && props.data.cfm25_calculator?.conditioned_floor_area ? ((props.data.cfm25_calculator.cfm25 / props.data.cfm25_calculator.conditioned_floor_area) * 100).toFixed(2) : null } </p>
+            </ShowOrHide>
+            
+            <ShowOrHide visible={props.data?.duct_leakage_test_method === 'Pressure pan'}> 
+                <Photo id="pressure_pan_setup_photo" label="Pressure Pan Setup – Photo">
+                    Take a photo of the pressure pan and manometer measuring one supply register.
+                </Photo>   
+                <Photo id="pressure_pan_results_photo" label="Pressure Pan Results – Photo">
+                    Take a photo of the document showing the readings from all the supply registers.
```
[the remark this answers is not in the document]
Comment (sudhacheran) [reply src/templates/doe_workflow_central_ducted_split_heat_pump.mdx]: Done

File: src/templates/doe_workflow_central_ducted_split_heat_pump.mdx [hunk 3]
```diff
@@ -314,23 +335,47 @@
...
+                </Photo>        
+            </ShowOrHide>
             
+            <ShowOrHide visible={props.data?.duct_leakage_test_method === 'Airflow hood (capture hood, balometer)'}> 
+                <Photo id="airflow_hood_setup_photo" label="Airflow Hood Setup  – Photo">
+                    Take a photo of the airflow hood measuring one supply register.
```
[the remark this answers is not in the document]
Comment (sudhacheran) [reply src/templates/doe_workflow_central_ducted_split_heat_pump.mdx]: Done

File: src/templates/doe_workflow_central_ducted_split_heat_pump.mdx [hunk 4]
```diff
@@ -314,23 +335,47 @@
...
+                </Photo>   
+                <Photo id="airflow_hood_results_photo" label="Airflow Hood Results  – Photo">
+                    Take a photo of the document showing the readings from all the supply registers.
```
[the remark this answers is not in the document]
Comment (sudhacheran) [reply src/templates/doe_workflow_central_ducted_split_heat_pump.mdx]: Done

File: src/templates/templates_config.ts
[the remark this answers is not in the document]
Comment (sudhacheran) [reply src/templates/templates_config.ts:1]: Have included IRA workflows separately. Edward's version has the show/hide component in the Heat Pump Ducted workflow.

File: src/templates/doe_workflow_central_ducted_split_heat_pump.mdx [hunk 5]
```diff
@@ -314,23 +341,47 @@
...
-                : null
-            } 
             
-            <p><strong>CFM25 =</strong>{props.data.cfm25_calculator?.cfm25 ? props.data.cfm25_calculator.cfm25 : null}</p>
-            <p><strong>Conditioned Floor Area (ft<sup>2</sup>) = </strong>{props.data.cfm25_calculator?.conditioned_floor_area ? props.data.cfm25_calculator.conditioned_floor_area : null} </p>
-            <p><strong>Duct CFM25 per 100 per ft<sup>2</sup> = </strong>{ props.data.cfm25_calculator?.cfm25 && props.data.cfm25_calculator?.conditioned_floor_area ? ((props.data.cfm25_calculator.cfm25 / props.data.cfm25_calculator.conditioned_floor_area) * 100).toFixed(2) : null } </p>
+            <p><strong>Test method used: </strong> {props.data?.duct_leakage_test_method}</p> 
+            <ShowOrHide visible={props.data?.duct_leakage_test_method === 'Duct leakage tester'}> 
+                <Photo id="duct_test_setup_photo" label="Duct Test Setup – Photo">
+                    Photo of duct test setup showing ring size and how it is attached to the duct system.
```
Comment (charliepnnl) [inline src/templates/doe_workflow_central_ducted_split_heat_pump.mdx:349]: Text should not have a period

**Review by charliepnnl — APPROVED** [review]

**Review by charliepnnl — APPROVED** [review]

**Pull request merged**